src/ocm/components/clusterWizard/wizardTransition.ts

```diff
@@ -91,6 +91,19 @@ const hostDiscoveryStepValidationsMap: WizardStepValidationMap = {
   softValidationIds: [],
 };
 
+const operatorsStepValidationsMap: WizardStepValidationMap = {
```

Comment:
There is already an empty WizardStepValidationMap. I would define this as a constant to avoid repetition etc.
And just to avoid issues in case this data is mutated somehwere, I would create a function that creates this object.

Eg.
```
const buildEmptyValidationMap = () => return object;
const staticIpValidationsMap: WizardStepValidationMap = buildEmptyValidationMap();
const operatorsStepValidationsMap: WizardStepValidationMap = buildEmptyValidationMap();
```